lib/cylc/gui/app_gcylc.py

```diff
@@ -656,10 +646,9 @@ def setup_views(self):
         num_views = 2
         self.view_containers = []
         self.current_view_toolitems = []
-        for i in range(num_views):
-            self.current_views.append(None)
-            self.view_containers.append(gtk.HBox())
-            self.current_view_toolitems.append([])
+        self.current_views += [None] * num_views
+        self.view_containers += [gtk.HBox()] * num_views
+        self.current_view_toolitems += [[]] * 3
```

Comment:
Argh - I'm on a roll this week!   No idea how I did that (I blame rushed trip prep) - currently transiting Melbourne airport, I'll fix this in the Northern Hemisphere...)